lib/features/pfis/pfi_page.dart

```diff
@@ -14,139 +16,182 @@ class PfiPage extends HookConsumerWidget {
+            ),
           ],
         ),
       ),
     );
   }
 
-  Widget _buildHeader(BuildContext context, String title, String subtitle) =>
-      Padding(
-        padding: const EdgeInsets.symmetric(
-          horizontal: Grid.side,
-          vertical: Grid.xs,
-        ),
-        child: Column(
-          children: [
-            Align(
-              alignment: Alignment.topLeft,
-              child: Text(
-                title,
-                style: Theme.of(context).textTheme.headlineSmall?.copyWith(
-                      fontWeight: FontWeight.bold,
-                    ),
-              ),
+  Widget _buildHeader(
+      BuildContext context, WidgetRef ref, String title, String subtitle) {
+    return Padding(
+      padding:
+          const EdgeInsets.symmetric(horizontal: Grid.side, vertical: Grid.xs),
+      child: Column(
+        children: [
+          Align(
+            alignment: Alignment.topLeft,
+            child: Text(
+              title,
+              style: Theme.of(context).textTheme.headlineSmall?.copyWith(
+                    fontWeight: FontWeight.bold,
+                  ),
             ),
-            const SizedBox(height: Grid.xs),
-            Align(
-              alignment: Alignment.topLeft,
-              child: Text(
-                subtitle,
-                style: Theme.of(context).textTheme.bodyMedium,
-              ),
+          ),
+          const SizedBox(height: Grid.xs),
+          Align(
+            alignment: Alignment.topLeft,
+            child: Text(
+              subtitle,
+              style: Theme.of(context).textTheme.bodyMedium,
             ),
-          ],
-        ),
-      );
-
-  Widget _buildPfiList(
-    BuildContext context,
-    WidgetRef ref,
-    ValueNotifier<Pfi?> selectedPfi,
-  ) =>
-      ref.watch(pfisProvider).when(
-            loading: () => const Center(child: CircularProgressIndicator()),
-            error: (error, _) => _buildError(context, ref, error),
-            data: (pfis) => ListView(
-              children: pfis
-                  .map(
-                    (pfi) => ListTile(
-                      title: Text(
-                        pfi.name,
-                        style: Theme.of(context).textTheme.bodyMedium?.copyWith(
-                              color: Theme.of(context).colorScheme.primary,
-                            ),
-                      ),
-                      subtitle: Text(pfi.didUri),
-                      trailing: (selectedPfi.value == pfi)
-                          ? Icon(
-                              Icons.check,
-                              color: Theme.of(context).colorScheme.primary,
-                            )
-                          : null,
-                      onTap: () async {
-                        selectedPfi.value = pfi;
+          ),
+          const SizedBox(height: Grid.xs),
+          Align(
+            alignment: Alignment.topLeft,
+            child: ElevatedButton(
```

Comment:
nit: instead of an `ElevatedButton` here, let's use a similar style to what's in `SendDidPage` [here](https://github.com/TBD54566975/didpay/blob/69d54dc716b9e99412f3001af70e6867b7f86a21/lib/features/send/send_did_page.dart#L138-L160)!